packages/esm-patient-banner-app/src/config-schema.ts

```diff
@@ -24,11 +24,33 @@ export const configSchema = {
     _description: "Whether to use the relationship name as a link to the person's patient chart",
     _default: false,
   },
+  showPrintIdentifierStickerButton: {
+    _type: Type.Boolean,
+    _description: "Whether to display the 'Print identifier sticker' button in the patient banner",
+    _default: false,
+  },
+  printIdentifierStickerFields: {
+    _type: Type.Array,
+    _description: 'Patient demographics to include in the identifier sticker printout',
+    _default: ['name', 'dateOfBirth', 'age', 'gender', 'identifier'],
+    _elements: {
+      _type: Type.String,
+    },
+  },
+  printIdentifierStickerSize: {
+    _type: Type.String,
+    _description:
+      'Specifies the paper size for printing the sticker. You can define the size using units (e.g., mm, in) or named sizes (e.g., "148mm 210mm", "A1", "A2", "A4", "A5").',
+    _default: '4in 6in',
+  },
 };
 
 export interface ConfigObject {
   contactAttributeTypes: Array<string>;
   excludePatientIdentifierCodeTypes: Array<String>;
   customAddressLabels: Object;
   useRelationshipNameLink: boolean;
+  showPrintIdentifierStickerButton: boolean;
+  printIdentifierStickerFields: Array<String>;
+  printIdentifierStickerSize: String;
```

Comment:
```suggestion
  excludePatientIdentifierCodeTypes: Array<string>;
  customAddressLabels: Object;
  useRelationshipNameLink: boolean;
  showPrintIdentifierStickerButton: boolean;
  printIdentifierStickerFields: Array<string>;
  printIdentifierStickerSize: string;
```

https://www.typescriptlang.org/docs/handbook/basic-types.html#about-number-string-boolean-symbol-and-object